tests/player/licenses.test.ts

```diff
@@ -36,7 +42,7 @@ describe('player', () => {
 
     describe('update', () => {
       assertItCallsCorrectUrl('PUT', '/v1/player/licenses/my-license-id', () =>
-        licensesClient('my-license-id').update()
+        licensesClient('my-license-id').update({})
```

Comment:
does it fail now when not passing an empty object?

Reply:
no it does not fail, I have just fixed typescript error , as argument of that function can not be  `undefined` but it has to be `UpdatePlayerLicense`, I think this is a left over from somebody else from before